jans-cedarling/cedarling/src/authz/mod.rs

```diff
@@ -351,27 +351,17 @@ struct ExecuteAuthorizeParameters<'a> {
 }
 
 /// Structure to hold entites created from tokens
-#[derive(Clone)]
 pub struct AuthorizeEntitiesData {
     pub workload: Option<Entity>,
     pub user: Option<Entity>,
     pub resource: Entity,
     pub roles: Vec<Entity>,
     pub tokens: HashMap<String, Entity>,
+
+    pub build_entities: BuiltEntities,
```

Comment:
why the extra space here?
```suggestion
    pub build_entities: BuiltEntities,
```

Reply:
removed https://github.com/JanssenProject/jans/pull/10807/commits/78d57eee0a035f65ab046e436e806fa1a0c4644f